orchagent/vxlanorch.cpp

```diff
@@ -838,6 +1730,15 @@ bool VxlanTunnelMapOrch::delOperation(const Request& request)
         return true;
     }
 
+    auto vlan_id = (sai_vlan_id_t) vxlan_tunnel_map_table_[full_tunnel_map_entry_name].vlan_id;
+    if(!gPortsOrch->getVlanByVlanId(vlan_id, vlanPort))
+    {
+        SWSS_LOG_ERROR("Delete VLAN-VNI map.vlan id doesn't exist: %d", vlan_id);
+        return true;
+    }
+
+    vlanPort.m_vnid = (uint32_t) 0xFFFFFFFF;
+
     auto tunnel_map_entry_id = vxlan_tunnel_map_table_[full_tunnel_map_entry_name];
```

Comment:
`auto tunnel_map_entry_id = vxlan_tunnel_map_table_[full_tunnel_map_entry_name];` should be changed to 
`auto tunnel_map_entry_id = vxlan_tunnel_map_table_[full_tunnel_map_entry_name].map_entry_id;`

Reply:
Done